iina/DraggingDetect.swift

```diff
@@ -0,0 +1,179 @@
+//
+//  DraggingDetect.swift
+//  iina
+//
+//  Created by Yuze Jiang on 05/08/2017.
+//  Copyright © 2017 lhc. All rights reserved.
+//
+
+import Foundation
+
+extension PlayerCore {
+
+  func openURLs(_ urls: [URL]) -> Int? {
+    let paths = urls.map{ $0.path }
```

Comment:
I think, it's better to pass `URL`s rather than `String`s to the following methods. Some apparent benefits are:
- It's easier to check whether a `URL` is a directory, or getting path extension
- It's safer to append path components, like "/BDMV"